tools/pvacvector/run.py

```diff
@@ -112,8 +112,62 @@ def run_pipelines(input_file, base_output_dir, args, spacer):
 
     return parsed_output_files
 
+def write_min_scores(min_scores_rows, directory, args):
+    #This will write the junction scores for all tested spacers
+    min_scores_file = os.path.join(directory, '..', '..', 'junction_scores.tsv')
+    rows = []
+    with open(min_scores_file, 'w') as fh:
+        fieldnames = ['left_peptide', 'spacer', 'right_peptide', 'junction_score', 'epitope', 'allele', 'method']
+        writer = csv.DictWriter(fh, delimiter="\t", fieldnames=fieldnames)
+        writer.writeheader()
+        for row in min_scores_rows:
+            index_parts = row['Index'].split('|')
+            left_peptide = index_parts[0]
+            if len(index_parts) == 2:
+                spacer = 'None'
+                right_peptide = index_parts[1]
+            else:
+                spacer = index_parts[1]
+                right_peptide = index_parts[2]
+            new_row = {
+                'left_peptide': left_peptide,
+                'spacer': spacer,
+                'right_peptide': right_peptide,
+                'epitope': row['MT Epitope Seq'],
+                'allele': row['HLA Allele'],
+            }
+            if args.top_score_metric == 'lowest':
+                new_row['junction_score'] = float(row['Best MT Score'])
+                new_row['method'] = row['Best MT Score Method']
+            elif args.top_score_metric == 'median':
+                new_row['junction_score'] = float(row['Median MT Score'])
+                new_row['method'] = 'NA'
```

Comment:
instead of NA, would it be better to fill this with "median"?

Reply:
Sure. Do you think they columns are ok the way they are or is there any info you think would be good to also display?

Reply:
I think this is probably the right level of granularity. We can always parse the tmp epitope files if we really need additional info